units-tracking-app/units-tracking-app/BusinessLayer/ GoalsService.swift

```diff
@@ -0,0 +1,47 @@
+//
+//   GoalsService.swift
+//  units-tracking-app
+//
+//  Created by Maryia Karpava on 16/08/2023.
+//
+
+import Foundation
+
+
+class  GoalsService: ObservableObject {
+    func getUnitsPerDay() -> Int {
+        return 3
+    }
+    
+    func getUnitsPer7Days() -> Int {
```

Comment:
TODO: add documentation comments here and in the other method. (e.g. 3 units per day -- is 3 already a violation, or 3 is max value but still within a limit?)